<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="openBtn">
          <property name="text">
           <string>打开图像</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="turnBtn">
          <property name="text">
           <string>翻转图像</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disposeBtn">
          <property name="text">
           <string>边缘检测</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="grayBtn">
          <property name="text">
           <string>灰度化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_former">
          <property name="title">
           <string>原图</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGraphicsView" name="formerView"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_res">
          <property name="title">
           <string>生成图</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGraphicsView" name="resultView"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1294</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="separator"/>
    <addaction name="actionsave"/>
    <addaction name="actionsaveas"/>
   </widget>
   <widget class="QMenu" name="menu_image">
    <property name="title">
     <string>图像</string>
    </property>
    <addaction name="action_Gray"/>
    <addaction name="action_Binary"/>
    <addaction name="action_Hist"/>
    <addaction name="action_filter"/>
    <addaction name="action_Dct"/>
    <addaction name="action_Sobel"/>
   </widget>
   <widget class="QMenu" name="menu_video">
    <property name="title">
     <string>视频</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_image"/>
   <addaction name="menu_video"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionopen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="iconText">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>保存生成文件</string>
   </property>
  </action>
  <action name="action_Gray">
   <property name="text">
    <string>灰度化</string>
   </property>
  </action>
  <action name="action_Binary">
   <property name="text">
    <string>二值化</string>
   </property>
  </action>
  <action name="action_Hist">
   <property name="text">
    <string>直方图及均衡化</string>
   </property>
  </action>
  <action name="action_filter">
   <property name="text">
    <string>滤波</string>
   </property>
  </action>
  <action name="action_Dct">
   <property name="text">
    <string>离散余弦变换</string>
   </property>
  </action>
  <action name="actionsaveas">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_Sobel">
   <property name="text">
    <string>边缘检测</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
